src/main/cpp/t06_homework.cpp

```diff
@@ -80,9 +80,38 @@
 
 #include "t06_homework.h"
 #include <iostream>
-
+#include <set>
 using namespace std;
 
 int t06_homework() {
+int D,F = 0;
+cin >> D;
+string S1,S2;
+set<string> MSET;
+    for (int L = 0; L < D; L++) {
+    cin >> S1;
+    MSET.insert(S1);}
+ while (cin >> S2) {
+  bool CSET;
+  CSET =MSET.count(S2);
+  int CT = 1,CF = 0;
 
+    if (CSET)
+    {   CT = 0;}
+    else {char Conv;
+     for (int L = 0;L < S2.length();L++) {
+        Conv = S2[L];
+    if (L > '64' && L < '92')
+        {CF++;}}}
+        if (CF == 1) {
+        for (int L = 0; L < S2.length();L++)
+        {L = tolower(L);}
+        for (int L = 0; L < S2.length();L++) {
+        S2[L] = toupper(S2[L]);
+        if (MSET.count(S2))
+        {CT = 1; break;}
+        S2[L] = tolower(S2[L]);}
+        CT = 0;}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'CT' is reassigned a value before the old one has been used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4512320)